proxy/src/http/session.rs

```diff
@@ -80,18 +175,19 @@ mod test {
     use warp::test::request;
 
     use crate::registry;
+    use crate::session;
 
     #[tokio::test]
     async fn get() {
         let tmp_dir = tempfile::tempdir().unwrap();
         let registry = registry::Registry::new(radicle_registry_client::Client::new_emulator());
         let store = kv::Store::new(kv::Config::new(tmp_dir.path().join("store"))).unwrap();
-        let api = super::get_filter(
+        let api = super::filters(
             Arc::new(RwLock::new(registry)),
             Arc::new(RwLock::new(store)),
         );
 
-        let res = request().method("GET").path("/session").reply(&api).await;
+        let res = request().method("GET").path("/").reply(&api).await;
```

Comment:
Is session now on `/`?

Reply:
Nope, it's mounted under "/session" in the `routes` function.